packages/eslint-plugin/src/rules/no-invalid-void-type.ts

```diff
@@ -127,6 +127,35 @@ export default util.createRule<[Options], MessageIds>({
       }
     }
 
+    /**
+     * @brief checks if the generic type parameter extends void
+     */
+    function checkExtendedVoid(
```

Comment:
Nit: this isn't checking _extension_, no?

```suggestion
    /**
     * @brief checks if the generic type parameter defaults to void
     */
    function checkDefaultVoid(
```

Reply:
You're right!
I fixed!
https://github.com/typescript-eslint/typescript-eslint/pull/5671/commits/a1421824ec0864690a1e6ebf3750e0e75e39096d